Resources/views/content_fields.html.twig

```diff
@@ -13,3 +13,13 @@
         {% endif %}
     {% endspaceless %}
 {% endblock %}
+
```

Comment:
I'm afraid this is wrong. With this template you make global override for all object relation field types. Please remove it and use ez_render_field with proper view.